find-hardware.md

```diff
@@ -0,0 +1,188 @@
+# Finding your hardware
+
+This section is mostly a mini-guide on how to find what hardware you're currently running, this is mainly relevant for laptop and prebuilt users as hardware specs are a bit more difficult to obtain. You can skip this page  and head to [Creating the USB](./installer-guide/) if you already know what hardware you have.
+
+For this we'll assume you have Windows or Linux installed:
+
+* [Finding hardware using Windows](#finding-hardware-using-windows)
+* [Finding hardware using Linux](#finding-hardware-using-linux)
+
+## Finding Hardware using Windows
+
+For this we have mainly 2 options:
+
+* Windows' Built-in DeviceManager
+* [AIDA64](https://www.aida64.com/downloads)
+
+Due to the easier to use GUI, we recommend downloading AIDA64 and running this as it's much easier to grab specs. However we'll show you both methods for obtaining hardware specs.
+
+### CPU Model
+
+| AIDA64 | DeviceManager|
+| :--- | :--- |
+| ![](./images/finding-hardware-md/cpu-model-aida64.png) | ![](./images/finding-hardware-md/cpu-model-devicemanager.png) |
+
+### GPU Model
+
+| AIDA64 | DeviceManager|
+| :--- | :--- |
+| ![](./images/finding-hardware-md/GPU-model-aida64.png) | ![](./images/finding-hardware-md/GPU-model-devicemanager.png) |
+
+### Chipset Model
+
+| AIDA64 | DeviceManager|
+| :--- | :--- |
+| ![](./images/finding-hardware-md/chipset-model-aida64.png) | ![](./images/finding-hardware-md/chipset-model-devicemanager.png) |
+
+* Note: Intel SOC based CPUs will have the chipset and other features already on the same die instead of being dedicated chips. This means trying to detect the exact chipset is a bit more difficult
+
+### Keyboard, Trackpad and Touchscreen Connection Type
+
+| DeviceManager |
+| :--- |
+| ![](./images/finding-hardware-md/trackpad-model-devicemanager.png) |
+
+AIDA64 unfortunately doesn't provide any useful info regarding pointer devices, so we recommend using DeviceManager for this.
+
+* You can find these devices under the following:
+  * `Human Interface Devices`
+  * `Keyboards`
+  * `Mice and other Pointer Devices`
+
+* To view the exact connection type of the device, select the pointer device then enter `View -> Device by Connection`. This will clarify whether it's over PS2, I2C, SMBus, USB, etc
```

Comment:
```suggestion
* To view the exact connection type of the device, select the pointer device then enter `View -> Devices by connection`. This will clarify whether it's over PS2, I2C, SMBus, USB, etc.
```